.github/workflows/npm-audit.yml

```diff
@@ -0,0 +1,30 @@
+name: NPM Audit Scan for Vulnerabilities
+run-name: Running an Audit Scan with NPM
+on:
+  push:
+
+jobs:
+  npm-audit-scan-job:
+    name: NPM Audit Scan for Vulnerabilities
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v4
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.x"
+
+      - name: Step3 Install Dependencies
+        run: npm install
+
+      - name: NPM Audit Scan and Generate Report in JSON Format
+        run: npm audit --json > npm-audit-results.json
```

Comment:
Same note about using `yarn`. You can add `yarn` to the beginning of your run statement.

Reply:
Added